common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -651,7 +688,10 @@ function (Sjson, AsyncProcess) {
         */
         updatePlayTime: function (time) {
             var state = this.state,
-                newIndex;
+                newIndex,
```

Comment:
See note above about either adding new declarations above the last item or into their own declaration statements.